### PR TITLE
fix: refine dragging view area

### PR DESCRIPTION
**Pull request by btea**
When dragging the view area to view the responsive effect, since the width calculation percentage is now an integer, some edge case values will be skipped directly, making it difficult to meet the conditions you want to test.

### CODE REVIEW

File: src/SplitPane.vue [hunk 1]
```diff
@@ -37,7 +37,7 @@ function dragMove(e: MouseEvent) {
       ? container.value.offsetHeight
       : container.value.offsetWidth
     const dp = position - startPosition
-    state.split = startSplit + ~~((dp / totalSize) * 100)
+    state.split = startSplit + Number(((dp / totalSize) * 100).toFixed(2))
```
Comment (sxzz) [inline src/SplitPane.vue:40]:
```suggestion
    state.split = startSplit + +((dp / totalSize) * 100).toFixed(2)
```

**Review by sxzz — APPROVED** [review]
